tests/virtualization/universal/windows.pm

```diff
@@ -0,0 +1,42 @@
+#Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: Import and test Windows guest
+# Maintainer: Pavel Dostal <pdostal@suse.cz>
+
+use base "consoletest";
+use virt_autotest::common;
+use virt_autotest::utils;
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub remove_guest {
+    my $guest = shift;
+
+    if (script_run("virsh list --all | grep '$guest'", 90) == 0) {
+        assert_script_run "virsh destroy $guest";
+        assert_script_run "virsh undefine $guest";
+    }
+}
+
+sub run {
+    my $self = shift;
+
+    remove_guest $_ foreach (keys %virt_autotest::common::imports);
+
+    import_guest $_, 'virt-install' foreach (values %virt_autotest::common::imports);
+    add_guest_to_hosts $_, $virt_autotest::common::imports{$_}->{ip} foreach (keys %virt_autotest::common::imports);
+
+    # Check if SSH is open because of that means that the guest is installed
+    ensure_online $_ foreach (keys %virt_autotest::common::imports);
```

Comment:
LGTM overall. 
Just a small comments, is that possible login in the system and check system version ? But this maybe need more effort to create new windows needle etc..

Reply:
Your wish is my command! See [here](http://pdostal-server.suse.cz/tests/9963#step/windows/26)